examples/smbserver.py

```diff
@@ -54,6 +54,7 @@
 
     server.addShare(options.shareName.upper(), options.sharePath, comment)
     server.setSMB2Support(options.smb2support)
+    server.setSMBChallenge('') #here you can set a custom smb challenge, if empty defaults to 'A'*8 (remember: must be 8 bytes (16 chars) long)
```

Comment:
I changed the comment a lil bit to make more clear. Let me know if you're ok with it and please change it (I also put it before the actual code):

    # Here you can set a custom SMB challenge in hex format
    # If empty defaults to '4141414141414141'
    # (remember: must be 16 hex bytes long)
    # e.g. server.setSMBChallenge('12345678abcdef00') 
    server.setSMBChallenge('') 

grazie!